src/Symfony/Component/Mailer/Transport/SendmailTransport.php

```diff
@@ -114,7 +114,7 @@ protected function doSend(SentMessage $message): void
         $this->stream->setCommand($command);
         $this->stream->initialize();
         foreach ($chunks as $chunk) {
-            $this->stream->write($chunk);
+            $this->stream->write($chunk, false);
```

Comment:
For next reviewers: this sets the debug flag, so that the stream instance doesn't keep a string copy of what was written. This debug info is not accessible anywhere anyway.